### PR TITLE
Added db2host output var needed for db2schema module

### LINKED ISSUE #8: Add db2host var in output to be used by db2schema module

### PR DESCRIPTION
**Pull request by RamyaRaghuveera**
Signed-off-by: Ramya Raghuveera <ramragh1@in.ibm.com>

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -8,6 +8,7 @@ locals {
   yaml_dir      = "${path.cwd}/.tmp/${local.name}/chart/${local.name}"
   #db2instanceid    = timestamp()
   db2instanceid    = random_id.db2id.dec
+  db2host = "c-db2oltp-${local.db2instanceid}-db2u-engn-svc"
```
Comment (triceam) [inline main.tf]: @RamyaRaghuveera will all DB2 instances follow this format?

Comment (RamyaRaghuveera) [reply main.tf]: @triceam yes

Comment (triceam) [reply main.tf]: It looks like the prefix can change depending on the version of DB2.  For example: 

- `c-db2ucluster-sample-db2u-engn-svc` (from here): https://www.ibm.com/docs/en/cloud-paks/cp-data/4.0?topic=services-connecting-db2-openshift-tls-route
- `c-db2oltp-${local.db2instanceid}-db2u-engn-svc` for db2oltp deployments
- `c-db2wh-1605022957148004-db2u-engn-svc` for [db2wh](https://www.ibm.com/docs/en/cloud-paks/cp-data/4.0?topic=ps-retrieving-db2-port-number).

I would make the prefix a configuration variable, with a default value.

Comment (RamyaRaghuveera) [reply main.tf]: @triceam I have made both prefix and suffix as configuration variables

**Review by triceam — APPROVED** [review]

**Pull request merged**